packages/blockchain-link/tests/integration/solana.ts

```diff
@@ -0,0 +1,53 @@
+import BlockchainLink from '../../lib';
+import SolanaWorker, { SolanaAPI } from '../../lib/workers/solana';
+
+export const solanaApi = {
+    getGenesisHash: () => '5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d',
+    getVersion: () => ({ 'feature-set': 1879391783, 'solana-core': '1.14.22' }),
+    getParsedBlock: () => ({
+        blockHeight: 195138557,
+        blockTime: 1684668753,
+        blockhash: '9Grw2oA6XBQ499EmBKJfyjhwATojKGupGhjWQiCWhpxB',
+        parentSlot: 195138556,
+        previousBlockhash: '2K6J2Nw3z4P9SsGTq2n33YvqByrn826zjMsLqXz3f3Yh',
+        rewards: [],
+        transactions: [],
+    }),
+    getBlockHeight: () => 195138557,
+    rpcEndpoint: 'dummyUrl',
+} as unknown as SolanaAPI;
+
+describe(`Solana`, () => {
```

Comment:
For integration tests, solana is bit similar to `electrum` as it does not use the `BackendWebsocketServerMock`